style.css

```diff
@@ -0,0 +1,79 @@
+
+.title {
+	font-size: 10px;
+}
+
+body{ 
+	font-family:arial; 
+}
+
+.nav {
+	display: flex;
+	justify-content: flex-end; /* разместим справа */
+	background: #008aff;
+}
+
+.nav a {
+	color:#fff;
+	padding:15px 10px;
+	text-decoration:none;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/ad/pullRequest?prid=5498083)